.github/workflows/build+test+deploy.yml

```diff
@@ -54,6 +54,21 @@ jobs:
       with:
         name: binaries
         path: ./out/*.nupkg
+        
+  publishReleaseBinaries:
+    needs: [ packReleaseBinaries, testReleaseBinariesInDotNet8Container, testReleaseBinariesWithDotNet10 ]
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Restore tools
+      run: dotnet tool restore
+    - name: Download artifact
+      uses: actions/download-artifact@v4
+      with:
+        name: binaries
+        path: ./out/
+    - run: ls -l ./out/
```

Comment:
@webwarrior-ws i guess this line is just for testing, can remove now?

Reply:
Removed.